packages/opentelemetry-basic-tracer/src/BasicTracer.ts

```diff
@@ -75,17 +85,8 @@ export class BasicTracer implements types.Tracer {
       return NOOP_SPAN;
     }
 
-    // span context
-    const traceId = parentSpanContext
-      ? parentSpanContext.traceId
-      : randomTraceId();
-    const spanId = randomSpanId();
-
-    // TODO: create a real Span
-    const span = new NoopSpan({
-      traceId,
-      spanId,
-    });
+    options.parent = parentSpanContext;
+    const span = new Span(this, name, options, this._spanProcessor);
```

Comment:
Already done in separate PR https://github.com/open-telemetry/opentelemetry-js/pull/145, please add comments there (if any).